dist/marker-panel.js

```diff
@@ -150,19 +151,12 @@ Fliplet.InteractiveMap.component('marker-panel', {
           }
         }
       });
-      window.addEventListener('message', function (event) {
```

Comment:
why is this gone away?

Reply:
It seems to have been removed from `js/interface/marker-panel.js`

Reply:
@squallstar Yes, it was removed, because we now use Fliplet.Widget.onCancelRequest in this file `js/interface/map-panel.js` witch is working for all providers called in this component.
And it would return the fix for this issue Fliplet/fliplet-studio#3990